testsuite/integration-arquillian/pom.xml

```diff
@@ -73,6 +73,8 @@
         <migration.70.version>1.9.8.Final</migration.70.version>
         <migration.70.authz.version>2.2.1.Final</migration.70.authz.version>
```

Comment:
I believe this property and the profile `<id>test-70-authz-migration</id>` is no longer needed.